comm_config.yaml

```diff
@@ -5,14 +5,14 @@ communications:
     enabled: false
     channel: 'SLACK_CHANNEL'
     token: 'SLACK_API_TOKEN'
-    notiftype: short                           # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
+    notiftype: short                # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
   
   # Settings for Mattermost
   mattermost:
     enabled: false
-    url: 'MATTERMOST_SERVER_URL'                # URL where Mattermost is running. e.g https://example.com:9243
-    token: 'MATTERMOST_TOKEN'			# Personal Access token generated by BotKube user
-    team: 'MATTERMOST_TEAM'			# Mattermost Team to configure with BotKube 
+    url: 'MATTERMOST_SERVER_URL'    # URL where Mattermost is running. e.g https://example.com:9243
+    token: 'MATTERMOST_TOKEN'			  # Personal Access token generated by BotKube user
+    team: 'MATTERMOST_TEAM'			    # Mattermost Team to configure with BotKube 
```

Comment:
Please fix the comment indentation. Let's have a uniform indentation for all the comments

Reply:
done